profile/README.md

```diff
@@ -1,25 +1,18 @@
 <div align="center">
-    
-<h1>EddieHub</h1>
-
-<h2>Collaboration 1st, Code 2nd! </h2>
-
-<img src="https://user-images.githubusercontent.com/70807500/136845625-7addf1dd-ea2f-41b2-9c6b-f810f80acd07.gif" />
-
-<h6><img src="https://github.com/EddieHubCommunity.png" height="60" /> Meet Hubber! Our Mascot! 🤝 <img src="https://github.com/EddieHubCommunity.png" height="60" /></h6>
-
-<p> Here, right now on <b> your </b> screen, you are in the presence of the most inclusive <i> Open Source </i> community on the planet! </p>
-
-<p> This organisation is the home of <i> open source </i>! Make Pull Requests, open Issues, share ideas, help each other and make improvements! </p>
-
-<h2> This is a place welcome to ALL! 🎉</h2>
-
+    <h1>EddieHub - Collaboration 1st, Code 2nd!</h1>
+    <img src="https://user-images.githubusercontent.com/70807500/136845625-7addf1dd-ea2f-41b2-9c6b-f810f80acd07.gif" />
+    <h6>
+        <img src="https://github.com/EddieHubCommunity.png" height="60" /> Meet Hubber! Our Mascot! 🤝 <img src="https://github.com/EddieHubCommunity.png" height="60" />
+    </h6>
+    <p> Here, right now on <b> your </b> screen, you are in the presence of the most inclusive <i> Open Source </i> community on the planet! </p>
+    <p> This organisation is the home of <i> open source </i>! Make Pull Requests, open Issues, share ideas, help each other and make improvements!</p>
```

Comment:
I think this is a bug in the space before the exclamation.
![image](https://user-images.githubusercontent.com/51391473/171421683-7bacf6cd-89ad-4d7f-9b63-58a88c82ded5.png)

```suggestion
    <p> This organisation is the home of <i> open source</i>! Make Pull Requests, open Issues, share ideas, help each other and make improvements!</p>
```